### PR TITLE
Feat/external service

### PR DESCRIPTION
**Pull request by dragolea**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -376,7 +378,7 @@ export class MyRepository extends BaseRepositoryDraft<MyEntity> {
 
 #### create
 
-`(method) this.create(entry: Entry<T>) : Promise<InsertResult<T>>`.
+`(method) this.create(entry: Entry<T>) : Promise<boolean>`.
```
Comment (hakimio) [inline README.md:381]: @dragolea why did you change the return of this method? ☹

Comment (dragolea) [reply README.md:381]: That was a mistake to have the InsertResult there, boolean is better, i think I've increases the major but due to some issues with release I did not mentioned in the release ... 

I will add this.

Comment (hakimio) [reply README.md:381]: Why was it a mistake? It was working before.
How do I get the created entity `ID` now?

Comment (hakimio) [reply README.md:381]: Following was working before:
```ts
const result = await this.notificationRepo.create(notification);
const id = result.query.INSERT.entries[0].ID;
```

Comment (dragolea) [reply README.md:381]: The ID is the returned ID of the created Instance ?

Comment (dragolea) [reply README.md:381]: If it's like that, I think then it's better to have it back as it was before.

Comment (hakimio) [reply README.md:381]: Yes, it's the `ID` of the created entity.

Comment (hakimio) [reply README.md:381]: Can you revert the change?

Comment (dragolea) [reply README.md:381]: Ok, I will add it later as it was before,
Not sure what was the reason why I've changes it to boolean, but I had dooubts about it, so I will change it back...

Comment (hakimio) [reply README.md:381]: Thank you.

Comment (dragolea) [reply README.md:381]: Hi, 
I just took some time and tested this, somehow that InsertResult.query.INSERT.entries[0] is pointless, and I think that was the reason wny I've deleted it


On the right part of the image you can see the new instance which needs to be created...
On the left part of the image you see exactly the same but without the ID.

This look like morroring the instance before is created.

image

Comment (dragolea) [reply README.md:381]: and I think the actual ID created is somewhere else is on the results property which is an array which has **lastInsertRowid**
image

Comment (hakimio) [reply README.md:381]: I am using `UUIDs` instead of auto-incremented SQL ID and it works correctly for me.
I guess the `UUID` is generated by `@sap/cds` before creating the entry.

Comment (hakimio) [reply README.md:381]: In any case the result is still not a `boolean`.

Comment (dragolea) [reply README.md:381]: What's you suggestion ? 
Would it be useful to provide the **lastInsertRowid** maybe under another name 

Pratically **.create** will return an object { createdID : '23232' } something ?

Comment (dragolea) [reply README.md:381]: But somehow this works only when you explicitly use the UUID and not the aspect cuid.
Could you please share also the Entity for which you are creating the instance ?

Comment (dragolea) [reply README.md:381]: Correct when using the `cuid`, you get the generated UUID back, I will check this and see how I can do it better.
Currently we released a new decorator called `ExternalService()` which connects to an external service and that's behaving different than normal create from CAP

Comment (dragolea) [reply README.md:381]: I will create an issue for this change.

Comment (hakimio) [reply README.md:381]: Just looking at the PR, it looks good.
Tomorrow I'll try to update my project.
Thanks again for very quick fix 🙂
